src/gen9_avc_encoder.c

```diff
@@ -4642,152 +5008,1442 @@ gen9_avc_kernel_wp(VADriverContextP ctx,
                                             media_function,
                                             &media_object_walker_param);
 
-    return VA_STATUS_SUCCESS;
+    return VA_STATUS_SUCCESS;
+}
+
+
+/*
+sfd related function
+*/
+static void
+gen9_avc_set_curbe_sfd(VADriverContextP ctx,
```

Comment:
Just confused. i did not see the code changes in this function gen9_avc_set_curbe_sfd. why it is also showed in the patch?
Same as gen9_avc_send_surface_sfd/gen9_avc_kernel_sfd?